tutorcairn/templates/cairn/apps/superset/bootstrap/courseoverview.json

```diff
@@ -1909,7 +1909,7 @@
                 "offset": 0,
                 "params": "{\"remote_id\": 49, \"database_name\": \"admin\"}",
                 "schema": "openedx",
-                "sql": "SELECT user_id,\n  course_id,\n  time,\n  arrayJoin(range(floor(toUInt64(toUnixTimestamp(time) - 5)), ceil(toUInt64(toUnixTimestamp(time) + 5)))) AS timestamp\nFROM events\nWHERE event_source = 'browser'",
+                "sql": "SELECT user_id,\n  course_id,\n  time,\n  arrayJoin(range(floor(toUInt64(toUnixTimestamp(time))), ceil(toUInt64(toUnixTimestamp(time) + 120)))) AS timestamp\nFROM events\nWHERE course_id <> '' and user_id > 0",
```

Comment:
So we are ignoring browser entirely in this new query? Why is that? The timestamp change is understandable. 
Also, if we can add some visualizations (before/after), it would make it easier to understand the impact.

Reply:
We are now picking both events(browser and server) where we have course_id and user_id because some events have source "server"(e.g `edx.grades.problem.submitted`) which we missed previously.